venues/fixtures-al/venues.json

```diff
@@ -376,6 +376,60 @@
             "tap_list_last_update_time": null
         }
     },
+    {
+        "model": "venues.venue",
+        "pk": 27,
+        "fields": {
```

Comment:
I think the merge conflict failed you here. Isn't Chattahoochee already in the file?

Reply:
WTH? I did a pull before I pushed the PR. Might be easier for me to figure out on my end.

Reply:
Ah. I bet I didn't pull master...